### PR TITLE
[RLlib] Fix/remove some CI tests and many_ppo release test.

### PR DESCRIPTION
**Pull request by sven1977**
Fix/remove some CI tests and many_ppo release test.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/long_running_tests/workloads/many_ppo.py [hunk 1]
```diff
@@ -34,13 +35,12 @@
...
             "config": {
-                "framework": "torch",
-                "num_workers": 7,
+                "num_env_runners": 7,
                 "num_gpus": 0,
                 "num_sgd_iter": 1,
             },
             "stop": {
-                "timesteps_total": 1,
+                NUM_ENV_STEPS_SAMPLED_LIFETIME: 1,
```
Comment (simonsays1980) [inline release/long_running_tests/workloads/many_ppo.py:43]: Does this mean, we only make quick tests if the class can be built and the algorithm runs?

**Review by simonsays1980 — APPROVED** [review]
LGTM.

File: release/long_running_tests/workloads/many_ppo.py (hunk 1 above)
Comment (sven1977) [reply release/long_running_tests/workloads/many_ppo.py:43]: Yeah, I guess 🤷 . I think this is a conceptual test for tune to be able to handle large amounts of samples, and not a stress test.

Comment (simonsays1980) [reply release/long_running_tests/workloads/many_ppo.py:43]: Thanks for clarifying this :)

**Pull request merged**